docs/stencil-docs/installing-stencil-cli/live-previewing-a-theme.md

```diff
@@ -3,17 +3,21 @@
 <div class="otp" id="no-index">
 
 ### On This Page

```

Comment:
Please remove the spaces between each bullet and update to match heading name changes